resources/META-INF/plugin.xml

```diff
@@ -280,4 +280,10 @@
 
     <library.type implementation="io.flutter.sdk.FlutterPluginLibraryType"/>
   </extensions>
+  
+  <!-- Dart Plugin extensions -->
```

Comment:
For this, you'll want to edit `plugin_template.xml` and regen using `bin/plugin generate`.

Reply:
👍 Forgot about that.  Thanks!